airflow/operators/check_operator.py

```diff
@@ -215,8 +223,7 @@ def __init__(
         self.days_back = -abs(days_back)
         self.conn_id = conn_id
         sqlexp = ', '.join(self.metrics_sorted)
-        sqlt = ("SELECT {sqlexp} FROM {table}"
-                " WHERE {date_filter_column}=").format(**locals())
+        sqlt = "SELECT {sqlexp} FROM {table} WHERE {date_filter_column}=".format(**locals())
```

Comment:
Can we get rid of `**locals()` here and be explicit?

Reply:
Sure. :)